include/can/firmware/hal_can_message_buffer.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "can/firmware/hal_can_bus.hpp"
+#include "common/core/freertos_message_buffer.hpp"
+
+namespace hal_can_message_buffer {
+
+constexpr auto read_message_buffer_size = 1024;
```

Comment:
feels like this should be a defaulted template argument

Reply:
I don't think so. This is a specific message buffer. The one populated by ISR. I will add comments.